cypress/PageObjects/EditProfilePage.js

```diff
@@ -21,7 +21,12 @@ class EditProfilePage {
   }
 
   get saveButton() {
-    return cy.get('[role="button"]').eq(2)
+    return cy.xpath('[data-testid="save_button"]')
```

Comment:
Probably you should use .get instead ?
as here: https://github.com/GoodDollar/GoodDAPP/pull/1843/files#diff-d68e334a485565a9c547d9818cc6f0aaR19

I remeber, @sirpy have asked you do not use .xpath()

Reply:
yes, sure